src/components/Search/Search.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2023 New Vector Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import classnames from "classnames";
+import React from "react";
+import styles from "./Search.module.css";
+import { Field, Label } from "../Form";
+
+import SearchIcon from "@vector-im/compound-design-tokens/icons/search.svg";
+
+type SearchProps = {
+  /**
+   * The CSS class name
+   */
+  className?: string;
```

Comment:
This is the class name of the `Label` wrapped by `Field`. From a developer perspective it would be good to be able to set the container class as well.

What about adding this to the `Field` classes instead? Then you can tweak everything, if required.

Reply:
Using `asChild` instead so the field is merged with the label. That solves the extra DOM element issue.